src/dev/riscv/lupv.hh

```diff
@@ -47,6 +48,7 @@ class LupV : public Platform
   public:
     Clint *clint;
     LupioRTC *lupioRTC;
+    int lupioBLKIntID;
```

Comment:
This isn't used, right? Why do we need it?

Reply:
ping

Reply:
We use this to define the interrupt ID for the block device in both LupV.py and in our new board.

Reply:
But it's an unused variable, right? I grepped and I don't see where it's ever used.